src/types/enviroment.d.ts

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) 2022 Joshua Schmitt
```

Comment:
Could you remove your comment here? Because this source code is under GNU General Public License v3.0 as shown in https://github.com/ZynerOrg/xyter/blob/main/LICENSE. Also by signing your commits (git with gpg keys)  you basicly sign your work.

Reply:
Oh sorry it's my vscode addon I forgot to turn it off. As I said it's new to work with other people 😅

Reply:
I understand, no problem! Could you link to that extension? I may want to use it for some other projects :)

Reply:
[Here](https://marketplace.visualstudio.com/items?itemName=ymotongpoo.licenser) we go.

Reply:
Thank you 👍